### ISSUE TITLE
Rando Feature: Edge text to differentiate exits

### ISSUE THREAD

**Issue by Avasam** [Avasam/ptle-tools#73]
When a level has multiple exits, we still have to guess despite having access to the graph. Something like this would be really nice:
![image](https://github.com/user-attachments/assets/8d57a5f9-041b-41f9-addb-67fe677b4d5e)

(maybe optional as it can lead to clutter, we'll see)

**Comment by wossnameGitHub** [comment]
Yeah sure I can do that, easy as pie. Except for 2 problems that is:

Problem 1: yes this will lead to an extreme amount of clutter, not sure we want that
Problem 2: I'm not going to do this or anything else for that matter until all changes I've already made are merged
